.md/.project/security.project.md

```
@@ -0,0 +1,37 @@
+Security Policy for DevEnviro
+We take the security of our project seriously. If you discover a security vulnerability, we appreciate your help in disclosing it to us privately so we can address it before it becomes public knowledge.
+
+Supported Versions
+Since DevEnviro is under active development, only the latest version is supported. Security updates will be applied to the most recent commit on the main branch.
+
+Version	Supported
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 8

(MD010, no-hard-tabs)